src/amo/components/HeroRecommendation/index.js

```diff
@@ -233,7 +233,7 @@ export class HeroRecommendationBase extends React.Component<InternalProps> {
           // L10n: If uppercase does not work in your locale, change it to lowercase. This is used as a secondary heading.
           titleText = i18n.gettext('BY FIREFOX');
         } else {
-          titleText = i18n.gettext('SPONSORED');
+          titleText = i18n.gettext('PROMOTED');
```

Comment:
This should never happen (addons-server shouldn't allow it), but it's a safe error state - better than defaulting to one of the other groups like Recommended or By Firefox - and less complicated than trying to handle it at this point in the render.

Reply:
If it should not happen, let's add a sentry log when it does happen so at least we know something wrong is happening.

Reply:
we don't have sentry integration for frontend (it was way too noisy)

Reply:
Too bad.